index.html

```diff
@@ -703,15 +703,17 @@ <h2 class='informative'>Implementation Patterns</h2>
     </section>
 
     <section id="messaging-spi">
-      <h2>Messaging SPI</h2>
+      <h2>Messaging</h2>
       <section class="informative">
+	<h3>Introduction</h3>
         <p>
-          Non-normative: the messaging SPI defines when messages are emitted by a Fedora
+          This section defines when messages are emitted by a Fedora
```

Comment:
By removing the "Non-normative" text, a person reading this wouldn't know that this is non-normative language. If this is, indeed, non-normative language (and it _is_ non-normative language), then some marker for a human reader should be there.